resources/helm/development.yaml

```diff
@@ -694,14 +694,19 @@ mod-calendar:
     enabled: true
     maxUnavailable: 1
 mod-camunda:
-  extraEnvVars: []
+  extraEnvVars:
+    - name: SPRING_JERSEY_APPLICATIONPATH
```

Comment:
Is this value should be dynamic based on env it runs or it is static?

Reply:
@OHaimanov 

I suppose it is statis for now accordingly to information provided in related Jira ticket
https://folio-org.atlassian.net/browse/RANCHER-1875

![image](https://github.com/user-attachments/assets/ad2984e9-a3f1-4b3b-b24b-73117aa9230c)